src/examples/mnist/MNIST_SP.cpp

```diff
@@ -31,16 +31,32 @@
 #include <htm/algorithms/SpatialPooler.hpp>
 #include <htm/algorithms/SDRClassifier.hpp>
 #include <htm/utils/SdrMetrics.hpp>
+#include <htm/os/Timer.hpp>
 
 #include <mnist/mnist_reader.hpp> // MNIST data itself + read methods, namespace mnist::
 #include <mnist/mnist_utils.hpp>  // mnist::binarize_dataset
 
-namespace examples {
 
 using namespace std;
 using namespace htm;
 
 class MNIST {
+/**
+ * RESULTS:
+ *
+ * Order :	score				: column dimensions 	: #passes on train	: time to run (s)	: git commit	: comment
+ * -------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+ * 1/	Score: 97.11% (289 / 10000 wrong)	: 28x28x16		: 4			: 557			: 1f0187fc6 	: epochs help, at cost of time 
+ *
+ * 2/	Score: 96.56% (344 / 10000 wrong)	: 28x28x16		: 1			: 142			: 3ccadc6d6  
```

Comment:
competitive results, considering it's single pass, and many "ML" features missing (dropout, data-modification, cross-validation, param opt, many training passes...)

Reply:
I literally can not read this table because the lines are too long.

Reply:
better now?